tests/src/unit-readme.cpp

```diff
@@ -160,10 +160,10 @@ TEST_CASE("README" * doctest::skip())
             CHECK(foo == true);
 
             // other stuff
-            j.size();     // 3 entries
-            j.empty();    // false
-            j.type();     // json::value_t::array
-            j.clear();    // the array is empty again
+            CHECK(j.size() == 3);                     // 3 entries
+            CHECK_FALSE(j.empty());                   // false
+            CHECK(j.type() == json::value_t::array);  // json::value_t::array
```

Comment:
Are these comments needed?  They seem pretty redundant and could potentially become stale.

Reply:
:man_shrugging:

As Niels mentioned earlier, he just copy & pasted the README.

Reply:
Ah, I missed that the checks were part of the readme too.